src/panels/lovelace/components/hui-image.ts

```diff
@@ -193,8 +193,6 @@ export class HuiImage extends LitElement {
         style=${styleMap({
           paddingBottom: useRatio
             ? `${((100 * this._ratio!.h) / this._ratio!.w).toFixed(2)}%`
-            : !this._lastImageHeight
-            ? "56.25%"
```

Comment:
instead of removing this, we should check if `this._lastImageHeight` is `undefined` instead of falsy

Reply:
👍